src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -112,6 +112,7 @@
 		mutableAudioOrVideoContainers.put(FormatConfiguration.THREEGPP2, new AudioVariantInfo(new THREEG2A(), FormatConfiguration.THREEGA));
 		// XXX WEBM Audio is NOT MKA, but it will have to stay this way until WEBM Audio is implemented.
 		mutableAudioOrVideoContainers.put(FormatConfiguration.WEBM, new AudioVariantInfo(new MKA(), FormatConfiguration.WEBA));
+		mutableAudioOrVideoContainers.put(FormatConfiguration.WMV, new AudioVariantInfo(new WMA(), FormatConfiguration.WMA));
```

Comment:
AFAIK, WMV is only video/x-ms-wmv and no audio only.
So it shouldn't be considered as such.
Only ASF can be.